style.css

```diff
@@ -368,3 +368,92 @@ p {
   font-size: 15px;
   color: #253858;
 }
+
+.contact {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #6070ff;
+  border-top-left-radius: 10rem;
+}
+
+.contact-items {
+  align-self: center;
+  width: 100%;
+}
+
+.contact h2 {
+  color: #fff;
+  font-size: 40px;
+  line-height: 52px;
+  padding: 52px;
+  text-align: center;
+}
+
+.contact p {
+  color: #fff;
+  font-size: 20px;
+  text-align: center;
+  padding: 0 20px;
+}
+
+.form-flex {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-top: 20px;
+}
+
+.form-flex-item {
+  width: 90%;
+  margin: 10px;
+}
+
+.form-flex-item input {
+  width: 100%;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 24px;
+  color: #172b4d;
+  font-size: 17px;
+}
+
+.textarea {
+  background-color: white;
+  border-radius: 5px;
+}
```

Comment:
- [ ] [OPTIONAL] Good work applying styles to your `<textarea>` element ✔️. However, I noticed the font color of your textarea is almost white, kindly consider changing the color to a darker one to enhance the user experience of your form. 🤓

    > Hint to style the textarea placeholder:
    ```
    .textarea::placeholder {
        color: #000;
    }
    ```